packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -13,7 +13,12 @@ import {
 } from 'vscode-debugadapter';
 import { DebugProtocol } from 'vscode-debugprotocol';
 import {
+  LineBpsInTyperef,
```

Comment:
Since you called the next item a LineBreakpointInfo, could you also call this one LineBreakpointsInTyperef instead of LineBpsInTyperef?